apps/atqr/api/src/app/infra/email/mailer.service.test.ts

```diff
@@ -101,7 +102,7 @@ describe('Mailer', () => {
       expect(isValidEmail.valueOf).toThrow(Error);
     });
 
-    //   //  Invalid Mail
+    //  Invalid Mail
     it('should throw email error', async () => {
```

Comment:
The test name should specify the condition for the expected behavior. There's no indication here as to say in which case error will be thrown